src/handlers/package_handler.py

```
@@ -0,0 +1,226 @@
+        )
+        self.add_item(self.count_input)
+
+    async def on_submit(self, interaction: discord.Interaction):
+        try:
+            count = int(self.count_input.value)
+            if count < 0:
+                try:
+                    await interaction.response.send_message(
+                        "❌ Count must be 0 or greater!", ephemeral=True
+                    )
+                except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)